op-node/rollup/derive/system_config.go

```diff
@@ -112,7 +112,7 @@ func ProcessSystemConfigUpdateLogEvent(destSysCfg *eth.SystemConfig, ev *types.L
 			return NewCriticalError(errors.New("too many bytes"))
 		}
 		if rollupCfg.IsEcotone(l1Time) {
-			if err := eth.CheckEcotoneL1SystemConfigScalar(scalar); err != nil {
+			if err := eth.CheckL1SystemConfigScalar(scalar); err != nil {
```

Comment:
Need to update this for operator fee. Conflicts with existing holocene changes, so I'm leaving it for now.